libtock-sync/peripherals/syscalls/gpio_async_syscalls.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <libtock/peripherals/syscalls/gpio_async_syscalls.h>
```

Comment:
```suggestion
```

So we don't leak libtock in libtock-sync.

Reply:
It's there to avoid duplicating the `DRIVER_NUM_GPIO_ASYNC`, presumably for the same reason you have the same include here ( https://github.com/tock/libtock-c/pull/553/files#diff-28af97b25b50c3d619c7a3ba099e6a2ff6ebcf972c1fc84800fa932c3c1159d7R3 ) and elsewhere in #553.